Podcast/RecommendButton.swift

```diff
@@ -11,6 +11,8 @@ import UIKit
 class RecommendButton: UIButton {
     let buttonTitlePadding: CGFloat = 7
     
+    var numberOfRecommendations: Int = 0
```

Comment:
I don't think we should store this in the Button, we should just set it manually. It's gross but allows us to ensure correctness.

Reply:
agreed.